contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -51,13 +51,13 @@
         }"
       >
         <VLayout row class="px-3">
-          <IconButton
+          <KIconButton
             icon="collapseAll"
-            :text="$tr('collapseAllButton')"
+            :tooltip="$tr('collapseAllButton')"
             @click="collapseAll"
           />
           <VSpacer />
-          <IconButton
+          <KIconButton
             :disabled="!ancestors || !ancestors.length"
             icon="myLocation"
             :text="$tr('openCurrentLocationButton')"
```

Comment:
Just wanted to flag that this `:text` prop should be replaced with the `:tooltip` prop as well.

Reply:
Oh, right.
My bad for not noticing this earlier. Thanks!
Made the change.